src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");
+    }
+
+    public void stopWork() {
```

Comment:
The `stopWork` method should be annotated with `@Override` to maintain consistency and clarity that this method is overriding the superclass method.